backend/runner/plancheck/statement_report_executor_affected_rows.go

```diff
@@ -34,6 +37,43 @@ func getTableDataSize(metadata *storepb.DatabaseSchemaMetadata, schemaName, tabl
 	return 0
 }
 
+func buildGetTableDataSizeFunc(metadata *storepb.DatabaseSchemaMetadata) base.GetTableDataSizeFunc {
```

Comment:
Can we use `model.DatabaseMetadata` here instead of metadata? Currently, Bytebase always gets the `model.DatabaseMetadata` from database, we no need to use `model.NewDBSchema` here

Reply:
will do